src/main/DomEvents.mjs

```diff
@@ -262,12 +262,16 @@ class DomEvents extends Base {
             path = event.path;
         }
 
-        return {
-            path     : path.map(e => this.getTargetData(e)),
-            target   : this.getTargetData(event.target),
-            timeStamp: event.timeStamp,
-            type     : event.type
+        const result = {
+            path          : path.map(e => this.getTargetData(e)),
+            target        : this.getTargetData(event.target),
+            timeStamp     : event.timeStamp,
+            type          : event.type
+        }
+        if (event.relatedTarget) {
+            result.relatedTarget = this.getTargetData(event.relatedTarget);
```

Comment:
is the node itself sufficient, or should we also send the parent path?
this can be done like: `DomEvents.getPathFromElement(entry.target).map(e => DomEvents.getTargetData(e));`

Reply:
I would thunk the node is sufficient. We are going to need this when we start managing focus.